ts/features/itwallet/machine/eid/machine.ts

```diff
@@ -72,7 +72,11 @@ export const itwEidIssuanceMachine = setup({
           callbackUrl: event.authRedirectUrl
         }
       };
-    })
+    }),
+    setIsReissuing: assign(({ context }) => ({
+      ...context,
+      isReissuing: true
+    }))
```

Comment:
The spread operator is unnecessary in the `assign` actions
```suggestion
    setIsReissuing: assign(() => ({
      isReissuing: true
    }))
```